docs-src/simple-gateway.md

```diff
@@ -0,0 +1,40 @@
+# Deploying a simple Gateway
+
+The simplest possible deployment is a Gateway and Route resources which are deployed together and have the same owner. This represents the same kind of model used for Ingress. A Gateway is deployed and an HTTPRoute is deployed which matches all traffic and sends it to a single Service named foo-svc. 
+
+![Simple Gateway](single-service-gateway.png)
+
+```yaml
+kind: Gateway
+apiVersion: networking.x-k8s.io/v1alpha1
+metadata:
+  name: prod-web
+spec:
+  class: l7-gw-class
+  listeners:  
+  - protocol: HTTP
+    port: 80
+    routes:
+      kind: HTTPRoute
+      selector:
+        matchLabels:
+          gateway: prod-web
+```
+
+The Gateway represents the instantation of a logical load balancer. It's templated from a hypothetical `l7-gw-class` GatewayClass. The Gateway listens for HTTP traffic on port 80. This particular GatewayClass automatically assigns an IP address which will be shown in the Gateway.status after it has been deployed. 
+
+To match traffic that is sent to the Gateway listener an HTTPRoute can be bound to the Gateway. This is done using label selection from the Gateway to labels on the HTTPRoute.  The following HTTPRoute resource defines the mapping of a Gateway to a backend Service that traffic routes to. Because there are no host routes or paths specified, this HTTPRoute will match all traffic that arrives at port 80 of the load balancer and send it to the `foo-svc` Pods.
+
+```yaml
```

Comment:
I think this and the above example should ideally come from the examples folder so we can automatically validate them and ensure they don't become stale. Some of the other guides like [TLS](https://github.com/kubernetes-sigs/service-apis/blob/master/docs-src/tls.md) show this pattern being used.